src/main.cpp

```diff
@@ -147,7 +148,17 @@ Pinetime::System::SystemTask systemTask(spi,
                                         fs,
                                         touchHandler,
                                         buttonHandler);
+int mallocFailedCount = 0;
+int stackOverflowCount = 0;
```

Comment:
To be honest, I do not really like those global variables either. They are not strictly needed at runtime, but they provide useful information at development time (and also in the SystemInfo app).

Those variables could be wrapped into a object similar to other Components and passed to other modules by ref. I felt it was a bit over-engineered for just 2 variables, but it could be used for other purposes in the future. What do you think?

Reply:
I guess they could be moved into an anonymous namespace and then add new functions to just get the value. In fact a lot of the global objects are probably intended to be in an anonymous namespace.

Reply:
You're probably right about the global objects that should be in an anonymous namespace.
However, I don't think it's possible specifically for those 2 variables : that are used in an `extern "C"` scope (in main.cpp), and access using an `extern` declaration in SystemInfo.